ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -201,6 +202,18 @@ RUN git clone https://github.com/kanaka/noVNC /home/user/.novnc/ \
  && cd /home/user/.novnc \
  && npm install \
  && node ./utils/use_require.js --as commonjs --with-app
+ 
+# Install dependencies to enable codeintel (https://github.com/c9/c9.ide.language.codeintel)
+RUN virtualenv --python=python2 $HOME/.c9/python2
```

Comment:
Nit: Please hardcode `/home/user` instead of using the `$HOME` variable (just to be extra safe).

Reply:
Nit: All these lines need to be terminated by a `\` (otherwise `docker build` will complain about not knowing a `&&` Dockerfile instruction).